lib/mix/lib/mix/scm/git.ex

```diff
@@ -7,7 +7,7 @@ defmodule Mix.SCM.Git do
   end
 
   def format(opts) do
-    opts[:git]
+    "#{opts[:git]} (#{get_opts_rev(opts)})"
```

Comment:
Sorry, I didn't meant to say using `get_opts_rev` but rather `format_lock(opts)`. Note it may return `nil` though.

Reply:
But isn't this one enough for the purposes? I initially started with extracting the common code from inside format_lock, but then found this one. What is the advantage of using format_lock (note that i don't fully understand that method yet (or the overall system architecture for that matter) :)).

Reply:
Actually, using `format_lock` would lead to duplication on the `mix deps` output, so your approach is better. However, I am not a big fan of the double parens in the output. Maybe we should use this instead?

```suggestion
    "#{opts[:git]} - #{get_opts_rev(opts)}"
```

Reply:
Yeah, me neither. Agreed :)